### PR TITLE
Addisie esza

### PR DESCRIPTION
**Pull request by Addisie**

### CODE REVIEW

File: new/INS1022ESZA.xml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2024-12-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPlace>
+                <place type="privateHouse" subtype="institution">
+                    <placeName>Private collection of Qasis ʾƎšat Zammana</placeName>
```
Comment (CarstenHoffmannMarburg) [inline new/INS1022ESZA.xml:48]: I think, it is only useful to mark up places with `<placeName>`, if you want to give the a ref.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/INS1022ESZA.xml [hunk 2]
```diff
@@ -45,7 +45,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPlace>
                 <place type="privateHouse" subtype="institution">
-                    <placeName>Private collection of Qasis ʾƎšat Zammana</placeName>
+                    <placeName>Private collection of Qasis ʾƎšat Zammana ʾAndābet</placeName>
```
Comment (CarstenHoffmannMarburg) [inline new/INS1022ESZA.xml:48]: I think, we write the title "qasis" with small letters and without capitalisation.

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
You probably have not noticed but the server is offline at the moment, you cannot merge anything (anything you merge when the server is offline will not make it to the betamasaheft.eu as the "door" is closed)

**Comment by eu-genia** [comment]
updated

**Comment by Addisie** [comment]
Thank you so much!

**Pull request merged**